src/morpho-chainlink/interfaces/IMorphoChainlinkOracleV2Factory.sol

```diff
@@ -22,6 +22,7 @@ interface IMorphoChainlinkOracleV2Factory {
     /// - Feeds are either Chainlink-compliant or the address zero.
     /// - Feeds have the same behavioral assumptions as Chainlink's.
     /// - The vaults, if set, are ERC4626-compliant.
+    /// - The feeds, if set, are Chainlink-interface-compliant.
```

Comment:
We can remove line 22 now "Feeds are either Chainlink-compliant or the address zero."

Reply:
agree

Reply:
i removed also 
"Feeds have the same behavioral assumptions as Chainlink's."